README.md

```diff
@@ -232,6 +232,14 @@ Update screenshots for single package:
 yarn update:lumo --group vaadin-upload
 ```
 
+### Generating icons
+
+Re-generate SVG icons for packages that have them:
```

Comment:
```suggestion
Re-generate SVG icon sets and icon fonts from individual SVG files for the packages that have them (e.g. `vaadin-icons`):
```

Reply:
It's not clear when should this step be executed. Would it be possible to add a comment about that (e.g. "before releaseing", "after modifying any .svg files", etc)?

Improvement idea: Can this be an automated step in the release process? Can we remove the icon sets and the icon font out of the version control then?

Reply:
Good point, thanks. This script could be executed in CI and fail the build if `git diff` shows some files.

I will think about having a separate GitHub actions workflow for Lint step where we could add this.

Reply:
Updated README based on your suggestion.